Solutions/MarkLogicAudit/DataConnectors/Connector_MarkLogicAudit.json

```diff
@@ -0,0 +1,167 @@
+{
+    "id": "MarkLogic",
+    "title": "MarkLogicAudit",
```

Comment:
Provide space between Provider name and Appliance name.

Reply:
Done

Reply:
I am not able to see the proposed change in the file, can you check the file again.